test/ussd_clinic.test.js

```diff
@@ -955,19 +905,20 @@ describe("app", function() {
         });
 
         describe("state_id_type", function() {
+            var setupInputs = [
+                {session_event: 'new'}  // dial in
+                , "1"  // state_start - yes
+                , "1"  // state_consent - yes
+                , "123456"  // state_clinic_code
+                , "2"  // state_due_date_month - may
+                , "10"  // state_due_date_day
+            ];
+
             describe("choosing sa_id", function() {
                 it("should go to state_sa_id", function() {
                     return tester
                     .setup.user.addr("27820001001")
-                    .inputs(
-                        {session_event: 'new'}  // dial in
-                        , "1"  // state_start - yes
-                        , "1"  // state_consent - yes
-                        , "123456"  // state_clinic_code
-                        , "2"  // state_due_date_month - may
-                        , "10"  // state_due_date_day
-                        , "1"  // state_id_type - sa_id
-                    )
+                    .inputs.apply(this, setupInputs.concat('1'))
```

Comment:
Please add a comment here for what the response '1' is for (eg. `// state_id_type - sa_id` )